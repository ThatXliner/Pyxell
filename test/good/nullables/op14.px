
class A def

    func f(Int x: 0) Int def
        return x

    func g() def
        print 'g'


A? a = null
print a?.f()

a = A()
print a?.f(x=10)
a?.g()
