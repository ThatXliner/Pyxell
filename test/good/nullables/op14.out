null
10
g
